Cargo.toml

```diff
@@ -6,5 +6,8 @@ members = [
     'node',
     'runtime/virto',
     'runtime/kreivo',
-    'virto-xcm-emulator',
+]
+
+exclude = [
+    'virto-xcm-emulator'
```

Comment:
This was bringing issues when building the node with `--features runtime-benchmarks`. For the moment let's leave it like this since it does not change anything from the build perspective.